test/testSetup.ts

```diff
@@ -82,6 +83,7 @@ export class TestSetup {
       3
     );
     await browser.keys(['Window: Command Center']);
+    await utilities.pause(3);
```

Comment:
Pause is needed here to give the settings time to load after searching.  Prevents "element not interactable" error when the setting is not visible on the page.